guides/common/modules/proc_exporting-the-library-environment.adoc

```diff
@@ -19,7 +19,7 @@ To export the contents of the Library lifecycle environment of the organization,
 For more information, see xref:Download_Policies_Overview_{context}[].
 * Ensure that you synchronize Products that you export to the required date.
 
-.Export the Library Content of an Organization
+.Export the library content of an organization
```

Comment:
```suggestion
.Export the Library content of an organization
```
Don't we capitalize the ["Library" environment](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#capitalization)?

Reply:
Yes.